pkg/apis/duck/v1alpha1/subscribable_types.go

```diff
@@ -49,12 +49,40 @@ type SubscriberSpec struct {
 	// UID is used to understand the origin of the subscriber.
 	// +optional
 	UID types.UID `json:"uid,omitempty"`
+	// Generation of the origin of the subscriber with uid:UID.
+	// +optional
+	Generation int64 `json:"generation,omitempty"`
 	// +optional
 	SubscriberURI string `json:"subscriberURI,omitempty"`
 	// +optional
 	ReplyURI string `json:"replyURI,omitempty"`
 }
 
+// SubscribableStatus is the schema for the subscribable's status portion of the status
+// section of the resource.
+type SubscribableStatus struct {
```

Comment:
since this is really an array, subscribablestatuses?

Reply:
Kept it in sync with existing type - Subscribable (which contains an array). Do you still think we should make it plural - I am fine with either. Let me know